src/net_defs.h

```diff
@@ -98,20 +98,39 @@ struct _net_addr_s
     void *handle;
 };
 
-// magic number sent when connecting to check this is a valid client
+// Magic number sent when connecting to check this is a valid client
+#define NET_MAGIC_NUMBER     1454104972U
 
-#define NET_MAGIC_NUMBER 3436803284U
+// Old magic number used by Chocolate Doom versions before v3.0:
+#define NET_OLD_MAGIC_NUMBER 3436803284U
 
 // header field value indicating that the packet is a reliable packet
 
 #define NET_RELIABLE_PACKET (1 << 15)
 
+// Supported protocols. If you're developing a fork of Chocolate
+// Doom, you can add your own entry to this list while maintaining
+// compatibility with Chocolate Doom servers.
+typedef enum
+{
+    // Protocol introduced with Chocolate Doom v3.0. Each compatibility-
+    // breaking change to the network protocol will produce a new protocol
+    // number in this enum.
+    NET_PROTOCOL_CHOCOLATE_DOOM_0,
```

Comment:
how strong was the temptation to start the enum at 666? :)

Reply:
This actually has to start at 0; the values in this enum don't actually have any special value. Added some clarifying comments.

Oh, probably you mean the _0. I considered starting at _3 (for v3) but decided to be boring in the end.

Reply:
No I actually meant the enum value itself as a completely pointless easter egg (I thought you could start a C enumeration at a given int-space offset  by suffixing e.g. `= 1234` to the initial value, but I haven't double checked if I'm remembering correctly)